src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
 body {
+  height: 100vh;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$block-width: 100px;
+$block-height: 100px;
+$block-gap: 1px;
```

Comment:
Don't need to duplicate variables with the same values, also move variables to the separate file